packages/use-i18n/src/usei18n.tsx

```diff
@@ -93,8 +84,8 @@ const getCurrentLocale = ({
   return defaultLocale
 }
 
-type Context<Locale extends BaseLocale> = {
-  currentLocale: string
+type Context<LocaleParam extends BaseLocale, T extends string> = {
```

Comment:
What is T ?

Reply:
T is the union list of locale types supported. I will rename it